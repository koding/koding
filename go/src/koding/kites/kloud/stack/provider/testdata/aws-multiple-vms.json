{
	"provider": {
		"aws": {
			"access_key": "${var.aws_access_key}",
			"secret_key": "${var.aws_secret_key}"
		}
	},
	"resource": {
		"aws_instance": {
			"aws-instance": {
				"count": 3,
				"instance_type": "t2.nano",
				"tags": {
					"Name": "${var.koding_user_username}-${var.koding_group_slug}"
				},
				"user_data": "# let's create a file on your root folder:\necho \"hello world!\" >> /helloworld.txt\n# please note: all commands under user_data will be run as root.\n# now add your credentials and save this stack.\n# once vm finishes building, you can see this file by typing\n# ls /\n#\n# for more information please use the search box above"
			}
		}
	}
}
