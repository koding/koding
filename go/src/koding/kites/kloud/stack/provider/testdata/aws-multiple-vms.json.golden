{
	"provider": {
		"aws": {
			"access_key": "${var.aws_access_key}",
			"region": "eu-central-1",
			"secret_key": "${var.aws_secret_key}"
		}
	},
	"resource": {
		"aws_instance": {
			"aws-instance": {
				"ami": "ami-123456",
				"count": 3,
				"instance_type": "t2.nano",
				"key_name": "koding-kp",
				"security_groups": [
					"koding-sg"
				],
				"subnet_id": "koding-subnet",
				"tags": [
					{
						"Name": "${var.koding_user_username}-${var.koding_group_slug}"
					}
				],
				"user_data": "***"
			}
		}
	},
	"variable": {
		"kitekeys_aws-instance": {
			"default": {
				"0": "***",
				"1": "***",
				"2": "***"
			}
		}
	}
}